cmd/appgw-ingress/main.go

```diff
@@ -127,6 +126,14 @@ func validateNamespaces(namespaces []string, kubeClient *kubernetes.Clientset) {
 	}
 }
 
+func getKubeClient(apiConfig *rest.Config) *kubernetes.Clientset {
```

Comment:
are we using this ?

Reply:
No we are not - thanks for catching that!